static/favicon.svg

```diff
@@ -0,0 +1,7 @@
+<svg width="24" height="24" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Should be 0 0 24 24 (and content needs translate) or 0 1 24 24 to do the translate with viewbox.

Reply:
updated